src/fusiontrees/fusiontrees.jl

```diff
@@ -4,13 +4,18 @@
     struct FusionTree{I, N, M, L, T}
 
 Represents a fusion tree of sectors of type `I<:Sector`, fusing (or splitting) `N` uncoupled
-sectors to a coupled sector. (It actually represents a splitting tree, but fusion tree
-is a more common term). The `isdual` field indicates whether an isomorphism is present
-(if the corresponding value is true) or not. The field `uncoupled` contains the sectors
-coming out of the splitting trees, before the possible 𝑍 isomorphism. This fusion tree
-has `M=max(0, N-2)` inner lines. Furthermore, for `FusionStyle(I) isa GenericFusion`,
-the `L=max(0, N-1)` corresponding vertices carry a label of type `T`. If `FusionStyle(I)
-isa MultiplicityFreeFusion, `T = Nothing`.
+sectors to a coupled sector. It actually represents a splitting tree, but fusion tree
+is a more common term.
+
+## Fields
+- `uncoupled::NTuple{N,I}`: the uncoupled sectors coming out of the splitting tree, before the possible 𝑍 isomorphism.
+- `coupled::I`: the coupled sector.
+- `isdual::NTuple{N,Bool}`: indicates whether the an isomorphism is present (`true`) or not
```

Comment:
"the an isomorphism". The old text just had "an isomorphism". But that is not very descriptive. These are called Z isomorphisms in my convention, as present in the description of the `uncoupled` field. Maybe there you can add (see `isdual`).